apps/sms/index.html

```diff
@@ -113,6 +113,7 @@
     <script defer src="js/contact_renderer.js"></script>
     <script defer src="js/information.js"></script>
     -->
+    <link rel="import" href="waiting-screen.html">
```

Comment:
I'm wondering if we can use Fetch API instead of XHR [1]?

[1] https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API